pages/studentPortal/quizcomponent/edu-background.tsx

```diff
@@ -1,3 +1,49 @@
+<<<<<<< HEAD
+import { React, useState } from "react";
+import { Button } from "../../../components/ui/Button";
+
+const EducationBackground = () => {
+  const [eduBg, setEduBg] = useState({
+    institution: "",
+    degree: "",
+  });
+
+  const handleChange = (e) => {
+    const { name, value } = e.target;
+    setEduBg((prev) => {
+      return { ...prev, [name]: value };
+    });
+  };
```

Comment:
helpful to keep these functions in mind for now, but you can probably safely delete them for this PR. Definitely defer to what Raveen/Vithushan recommend if they disagree.